test/Common/HttpApiHandlerTests.cs

```diff
@@ -581,7 +581,7 @@ public async Task TerminateInstanceWebhook()
             var clientMock = new Mock<DurableOrchestrationClientBase>();
             clientMock
                 .Setup(x => x.TerminateAsync(It.IsAny<string>(), It.IsAny<string>()))
-                .Returns(Task.CompletedTask)
+                .Returns(Task.FromResult(true))

```

Comment:
Is this change still needed?

Reply:
Nope! I thought I missed something. It's weird that this didn't cause a build issue.